docs/learn/encyclopedia/contract-development/types/built-in-types.mdx

```diff
@@ -87,3 +88,11 @@ Address is a universal opaque identifier to use in contracts. It may represent a
 Address can be used as a contract function input argument (for example, to identify the payment recipient), as a data key (for example, to store the balance), as the authentication & authorization source (for example, to authorize a token transfer) etc.
 
 See [authorization documentation](../../security/authorization.mdx) for more details on how to use the `Address` type.
+
+## Option (`Option`)
+
+Option represents an optional value.
+
+The Option type is used for values that may, or may not, be present. The Option type is an enum and can either be `Some` (some value exists) or `None` (no value exits).
+
+While Option acts like Rust's Option<T>, the Option type is not explicitly represented in XDR. The Option type is represented in the XDR by the `ScVal` `ScVoid` value when no value exists (None), any by any other `ScVal` value when the value exists (Some).
```

Comment:
```suggestion
While Option acts like Rust's `Option<T>`, the Option type is not explicitly represented in XDR. The Option type is represented in the XDR by the `ScVal` `ScVoid` value when no value exists (None), any by any other `ScVal` value when the value exists (Some).
```

I think this is probably the main culprit. Docusaurus is expecting `<T>` to be a react component or something. Then, when it can't find a closing `</T>` it freaks out